aptos-move/framework/aptos-framework/sources/storage_gas.spec.move

```diff
@@ -15,17 +15,17 @@ spec aptos_framework::storage_gas {
         invariant max_gas <= MAX_U64 / BASIS_POINT_DENOMINATION;
         /// Invariant 3: The x-coordinate increases monotonically and the y-coordinate increasing strictly monotonically,
         /// that is, the gas-curve is a monotonically increasing function.
-        invariant (len(points) > 0 ==> points[0].x > 0)
-            && (len(points) > 0 ==> points[len(points) - 1].x < BASIS_POINT_DENOMINATION)
-            && (forall i in 0..len(points) - 1: (points[i].x < points[i + 1].x && points[i].y <= points[i + 1].y));
+        invariant (len(points) > 0 ==> points[0].x > 0) && (len(points) > 0 ==>
+            points[len(points) - 1].x < BASIS_POINT_DENOMINATION) && (forall i in 0..len(points)
+            - 1: (points[i].x < points[i + 1].x
```

Comment:
`len(points) - 1` should be in one line?

Reply:
A good discovery. Examples like this are essentially scenarios where spec syntax and forall loop syntax appear together, which are areas we already know we haven't done more tuning and testing on. For the example in the current comment, it can be achieved through syntax parsing to meet our human expectations, and it can be added to our to-do list for future optimization.

Reply:
We optimized it, you can see https://github.com/movebit/movefmt/blob/develop/tests/complex4/input1.fmt.move